src/app/components/Validators/ValidatorLink.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react'
+import { Link as RouterLink } from 'react-router-dom'
+import { useScreenSize } from '../../hooks/useScreensize'
+import Link from '@mui/material/Link'
+import { TrimLinkLabel } from '../TrimLinkLabel'
+import { RouteUtils } from '../../utils/route-utils'
+import Typography from '@mui/material/Typography'
+import { COLORS } from '../../../styles/theme/colors'
+import { Network } from '../../../types/network'
+
+type ValidatorLinkProps = {
+  address: string
+  name?: string
+  network: Network
+}
+
+export const ValidatorLink: FC<ValidatorLinkProps> = ({ address, name, network }) => {
+  const { isTablet } = useScreenSize()
+  const to = RouteUtils.getValidatorRoute(network, address)
+  return (
+    <Typography variant="mono" component="span" sx={{ color: COLORS.brandDark, fontWeight: 700 }}>
+      {isTablet ? (
+        <TrimLinkLabel label={address} to={to} />
```

Comment:
Don't we also want `name || address` here, instead of just `address` ?
(We do that for the non-tablet version a few lines below)